pkg/cmd/roachtest/tests/tpcc.go

```diff
@@ -908,25 +912,29 @@ func registerTPCC(r registry.Registry) {
 		Nodes: 3,
 		CPUs:  4,
 
-		LoadWarehousesGCE: 1000,
-		LoadWarehousesAWS: 1000,
-		EstimatedMaxGCE:   750,
-		EstimatedMaxAWS:   900,
+		LoadWarehousesGCE:   1000,
+		LoadWarehousesAWS:   1000,
+		LoadWarehousesAzure: 1000,
+		EstimatedMaxGCE:     750,
+		EstimatedMaxAWS:     900,
+		EstimatedMaxAzure:   900,
 
-		Clouds: registry.AllExceptAWS,
+		Clouds: registry.OnlyGCE,
```

Comment:
These tests can't run on local, so this should've been `OnlyGCE` in the first place.

Not sure if we want to run all the benchmarks on azure for cost reasons, but doing so would require kicking off a run to figure out better numbers for azure. The current numbers may or may not be high enough from my limited testing.